### PR TITLE
Utilize upstream events package for message publishing

### PR DESCRIPTION
**Pull request by tylerauerbeck**
- Removes namespaced mixin in favor of using metadata-api
- Cleans up Authconfig to pull values from AppConfig and utilize changes made in upstream echojwtx package
- Begins utilizing `events` package for publishing messages

### CODE REVIEW

File: cmd/serve.go [hunk 1]
```diff
@@ -137,20 +140,14 @@ func serve(ctx context.Context) error {
 	}
 
 	// jwt auth middleware
-	if !serveDevMode {
-		if authconfig, err := echojwtx.AuthConfigFromViper(viper.GetViper()); err != nil {
+	if viper.GetBool("oidc.enabled") {
```
Comment (rizzza) [inline cmd/serve.go:143]: isn't oidc enabled by default? Wouldn't we still want to be able to turn off a bunch of things via that `--dev` flag?

File: internal/graphapi/tools_test.go [hunk 2]
```diff
@@ -110,12 +108,17 @@ func setupDB() {
 		errPanic("failed to start nats server", err)
 	}
 
-	natsClient, err := newNatsClient(ns)
+	pubcfg := events.PublisherConfig{
+		URL:    ns.ClientURL(),
+		Prefix: "com.infratographer",
```
Comment (rizzza) [inline internal/graphapi/tools_test.go:113]: should the `Prefix` remain configurable via viper vs hard-coding here?

File: x/pubsubinfo/template/pubsub.tmpl [hunk 3]
```diff
@@ -225,21 +228,9 @@
...
 						}
 
-					msg := pubsubx.ChangeMessage{
-						EventType:    					eventType(m.Op()),
-						SubjectID:    					objID,
-						AdditionalSubjectIDs: 	additionalSubjects,
-						Timestamp: 							time.Now().UTC(),
-					}
-
+						fmt.Println(queueName)
```
Comment (rizzza) [inline x/pubsubinfo/template/pubsub.tmpl:231]: leftover debug?

File: internal/graphapi/tools_test.go (hunk 2 above)
Comment (tylerauerbeck) [reply internal/graphapi/tools_test.go:113]: I think it's fine to leave it hard-coded here as part of our tests.

Comment (rizzza) [reply internal/graphapi/tools_test.go:113]: didn't see that test filename there. My bad

File: x/pubsubinfo/template/pubsub.tmpl (hunk 3 above)
Comment (tylerauerbeck) [reply x/pubsubinfo/template/pubsub.tmpl]: ✅

**Review by rizzza — APPROVED** [review]

**Pull request merged**